src/main/java/org/matsim/episim/EpisimUtils.java

```diff
@@ -29,4 +29,27 @@ public static boolean hasStatusRelevantForInfectionDynamics(EpisimPerson personW
                 throw new IllegalStateException("Unexpected value: " + personWrapper.getDiseaseStatus());
         }
     }
+
+    public static boolean isPersonTraceable(EpisimPerson person){
+        //if person is in quarantine, it can not be tracked
+        if(! person.getQuarantineStatus().equals(EpisimPerson.QuarantineStatus.no)) return false;
+
+        switch (person.getDiseaseStatus()) {
+            case susceptible:
+                return true;
+            case infectedButNotContagious:
+                return true;
+            case contagious:
+                return true;
+            case seriouslySick:
+                return false; // assume is in hospital
+            case critical:
+                return false; // assume is in hospital
+            case recovered:
+                return true;
```

Comment:
Recovered persons will not be tracked, because they are filtered earlier.

Reply:
we now decided to split isRelevantForInfectionDynamics up again into 2 checks:
1) is the person actually on the go?
2) has the person a relevant disease status for infection dynamics?

That is to account for conditions for tracking (people have contact with each other) separately from conditions for infection (disease status must 'fit')